docs/controller-skills/milkrun.md

```diff
@@ -258,6 +265,22 @@ Use this information to know that you will be busy soon, and put out a request f
 - Aircraft not listening to your calls  
 It may already be too late! And to make matters worse, if you wait too long to ask, there might not be any controllers available to split the sector.
 
+### Pushback Requests on ACD
+During the event, the **SMC** at [YMML](../../aerodromes/classc/melbourne) controller may end up with a much higher workload than the **ACD** controller. Additionally, delays may need to be implemented for aircraft requesting pushback, so as to not overload the taxiways and holding points.
+
+To mitigate this, pushback requests may be done on **ACD** frequency, to balance the workload. A few steps must be followed to properly execute this procedure.
+
+1. **SMC** and **ACD** coordinate to implement the procedure, due to high **SMC** workload.
+2. **SMC** coordinates with **ADC** in order to have the [ATIS](../../aerodromes/classc/melbourne/#acd-pushback-requests) updated.
+3. When **ACD** has finished issuing an airways clearance, they will **remind** pilots to *"Contact me when ready for pushback"*.
+4. When a pilot requests pushback, **ACD** will instruct them to **Monitor** *(not contact)* Ground on 121.7.
```

Comment:
Flag to remind us to update this section based on the conversation above (the duplicated content on the ML ADC page)